site/_posts/2021-03-10-migrate-from-auth0.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog-post
+title: What to consider when you are considering an Auth0 migration
```

Comment:
What to think about when you're considering an Auth0 Migration

Reply:
Migrating off of Auth0? Here's what you need to know